node/src/components/small_network.rs

```diff
@@ -704,8 +714,10 @@ where
     }
 
     fn connect_to_peer_if_required(&mut self, peer_address: SocketAddr) -> Effects<Event<P>> {
+        self.blocklist
+            .retain(|_, ts| *ts > Timestamp::now() + *BLOCKLIST_RETAIN_DURATION);
```

Comment:
Won't the right-hand side will always be bigger? `ts` is the time when the entry was added, which has to be in the past, while the right side is `now() + RETAIN_DURATION`.  I guess we should remove `BLOCKLIST_RETAIN_DURATION` from `now()`, rather than add it.

Reply:
Thanks, @goral09, fixing this here https://github.com/CasperLabs/casper-node/pull/910